test/cache.js

```diff
@@ -0,0 +1,58 @@
+var test = require("tape")
+var webpack = require("webpack")
+var assign = require("object-assign")
+var conf = require("./utils/conf")
+var fs = require("fs")
+
+var cacheFilePath = "./node_modules/.cache/eslint-loader/data.json"
+
+test("eslint-loader can cache results", function(t) {
+
+  // delete the require cache for eslint-loader otherwise any previously run
+  // tests will have initialised the cache as false and prevent this test
+  // from creating the cache file
+  delete require.cache[require.resolve("../index.js")]
```

Comment:
I don't know if this is the best solution but all tests share the same instance of eslint-loader so without clearing the require cache, the cache option get initialised as false in previous tests and then it is impossible set it again